src/components/TimeTravel/TimeTravel.js

```diff
@@ -649,11 +805,39 @@ TimeTravel.propTypes = {
    * Optional callback handling timeline panning (e.g. for tracking)
    */
   onTimelinePan: PropTypes.func,
+  /**
+   * Enables Time Travel to be in the live auto-update mode
+   */
+  hasLiveMode: PropTypes.bool,
+  /**
+   * The live mode shows current time and ignores the `timestamp` param
+   */
+  showingLive: PropTypes.bool,
```

Comment:
I think you're right that it can be removed. Initially I thought of making the component more robust by allowing it to support live mode but not being there by default, but the implicit _stickiness_ would make it weird if we really wanted to use it that way.

I'll add a _TODO_ in the code, wouldn't risk removing it right now under the time pressure :)